src/main/java/bio/terra/workspace/service/get/GetService.java

```diff
@@ -0,0 +1,43 @@
+package bio.terra.workspace.service.get;
+
+import bio.terra.workspace.app.configuration.ApplicationConfiguration;
+import bio.terra.workspace.common.exception.SamApiException;
+import bio.terra.workspace.common.utils.SamUtils;
+import bio.terra.workspace.db.WorkspaceDao;
+import bio.terra.workspace.generated.model.WorkspaceDescription;
+import bio.terra.workspace.service.iam.AuthenticatedUserRequest;
+import bio.terra.workspace.service.iam.SamService;
+import org.broadinstitute.dsde.workbench.client.sam.ApiException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GetService {
+  private ApplicationConfiguration configuration;
```

Comment:
why is this one not final?

Reply:
it's actually unused so i'm just gonna remove it!